pointer/pointer.go

```diff
@@ -17,9 +17,9 @@ limitations under the License.
 package pointer
```

Comment:
Add a DEPRECATED note to the package comment and mention the alternative

Reply:
Done.